src/logic/basic.lean

```diff
@@ -477,6 +477,8 @@ by simp [and_comm]
 
 @[simp] theorem exists_eq {a' : α} : ∃ a, a = a' := ⟨_, rfl⟩
 
+@[simp] theorem exists_eq' {a' : α} : Exists (eq a') := ⟨_, rfl⟩
```

Comment:
```suggestion
@[simp] theorem Exists_eq {a' : α} : Exists (eq a') := ⟨_, rfl⟩
```
How about this name?

Reply:
It doesn't seem to be the convention. `Exists` should really be called `exists`